### PR TITLE
estimatr::tidy colnames are compatible with broom:tidy colnames

### PR DESCRIPTION
**Pull request by vincentarelbundock**
I was writing software that makes strong assumptions column names in the output of `broom::tidy`. It broke when I applied it to an `lm_robust` object, because `broom::tidy` uses `ci.low` and `ci.high`, while `estimatr::tidy` uses `ci.lower` and `ci.upper`. 

Since your function seems to have been inspired by `broom`, and since that package is under the popular tidyverse umbrella, I think it makes sense to standardize column names.

This commit just substitutes the names, and fixes the tests accordingly.

### CODE REVIEW

**Comment by acoppock** [comment]
Thank you so much for bringing this to our attention... we thought we'd matched `broom` but we were wrong! Say, are there different versions of broom, because I seem to get `conf.low` and `conf.high` not `ci.low`, `ci.high`

```
Y <- rnorm(100)
Z <- rnorm(100)

 tidy(lm(Y ~ Z), conf.int = TRUE)
         term    estimate  std.error  statistic    p.value   conf.low  conf.high
1 (Intercept)  0.03067781 0.09910207  0.3095577 0.75755459 -0.1659870 0.22734265
2           Z -0.18114862 0.09681666 -1.8710481 0.06432179 -0.3732781 0.01098089

tidy(estimatr::lm_robust(Y ~ Z))
         term    estimate  std.error    p.value   ci.lower     ci.upper df outcome
1 (Intercept)  0.03067781 0.09922599 0.75784775 -0.1662329  0.227588563 98       Y
2           Z -0.18114862 0.08957854 0.04587759 -0.3589143 -0.003382928 98       Y
```

**Comment by vincentarelbundock** [comment]
I don't think so. This test was created in 2014 and uses `conf.high`: https://github.com/tidyverse/broom/commit/9d85ac04586db86c9f01503f4e23aeb18efe53cb

I think your example has namespace issues. Try this:

```
> library(broom)
> library(estimatr)
> Y <- rnorm(100)
> Z <- rnorm(100)
> broom::tidy(lm(Y ~ Z), conf.int = TRUE)
         term    estimate  std.error  statistic   p.value   conf.low conf.high
1 (Intercept) -0.02296269 0.09470890 -0.2424555 0.8089340 -0.2109094  0.164984
2           Z  0.04446773 0.09457162  0.4702017 0.6392557 -0.1432066  0.232142

> estimatr::tidy(estimatr::lm_robust(Y ~ Z))
         term    estimate  std.error   p.value   ci.lower  ci.upper df outcome
1 (Intercept) -0.02296269 0.09650096 0.8124144 -0.2144657 0.1685403 98       Y
2           Z  0.04446773 0.10061243 0.6594843 -0.1551944 0.2441298 98       Y
```

**Comment by nfultz** [comment]
Please check that this doesn't break DeclareDesign::declare_estimator also.

**Comment by vincentarelbundock** [comment]
@nfultz I've never used DeclareDesign::declare_estimator, so I don't know what that would entail. Which test suit or example would you like me to run?

**Comment by nfultz** [comment]
Sorry Vincent, I meant that comment for luke or alex.

**Comment by acoppock** [comment]
Awesome, thanks @vincentarelbundock, looks like the pr just needs to be updated so that the change is to `conf.low` and `conf.high` rather than `ci.low` and `ci.high`.  Thanks for the reminder @nfultz .

File: R/estimatr_lm_robust.R [hunk 1]
```diff
@@ -149,7 +149,7 @@
 #' coef(lmro)
 #' tidy(lmro)$estimate
 #' # Can also get confidence intervals from object or with new 1 - `alpha`
-#' lmro$ci.lower
+#' lmro$conf.lower
```
Comment (lukesonnet) [inline R/estimatr_lm_robust.R]: `conf.low` not `conf.lower`

**Comment by vincentarelbundock** [comment]
I think this should do it. I squashed the commits to give you a cleaner PR merge. Hopefully I didn't mess anything up while doing so. 

Thanks for your work on this great package!

**Comment by vincentarelbundock** [comment]
@graemeblair good eye. I amended the existing commit: https://github.com/vincentarelbundock/estimatr/blob/65a400ad3ce6206d4a7fac4e7a98338eef2c9086/R/estimatr_iv_robust.R#L82

**Comment by lukesonnet** [comment]
@vincentarelbundock, I missed that you had changed the names of the `summary(lmrfit)$coefficients` matrix. Those are clobbered that way to be compatible with `summary(lmfit)$coefficients`, and so I'm reverting them back in the new PR #229. The output from `estimatr::tidy` will still be as per your changes.

**Comment by vincentarelbundock** [comment]
Sorry about that. Overaggressive use of `sed`

**Comment by lukesonnet** [comment]
Hah! Not your fault at all. I should have caught it in my review.

**Pull request merged**